src/codegen/doc/gi_docgen.rs

```diff
@@ -58,8 +59,9 @@ fn namespace_type_from_details(
     }
 }
 
-/// Convert a "Namespace.Type.method_name" to (Option<Namespace>, Option<Type>, name)
-/// Type is only optional for global functions and the order can be modified the `is_global_func` parameters
+/// Convert a "Namespace.Type.method_name" to (Option<Namespace>, Option<Type>,
+/// name) Type is only optional for global functions and the order can be
+/// modified the `is_global_func` parameters
```

Comment:
Backticks and intradoc links.

Reply:
those all were done automatically with rustfmt doc wrapping feature. Feel free to submit a patch fixing them if you want to :)

Reply:
That was expected as mentioned above, I still think you could've put more effort into at least reading the diff that you're submitting (see also the cases where this reformat _outright broke_ the comments). I don't care enough about `gir` to fix your changes, in part because of exactly this approach.

Reply:
Sure, I do plan to spend more time improving the whole docs situation. Whether things slightly broke now or not it is not a major problem as the whole docs situation is very bad.

Reply:
Sure, I hope the 26 comments here at least provide you a good starting-point for that, since not only docs broke but also standard comments.

Reply:
Yes, thank you for taking the time to carefully review all the changes. I will fix all of them & more in the upcoming days